scripts/34-Plain-Text-Team.md

```diff
@@ -0,0 +1,601 @@
+<style>
+:root {--r-code-font: "FiraCode Nerd Font";}
+.reveal .hljs {min-height: 50%;}
+</style>
+
+![[git-logo.png|500]]
+
+# Plain-Text Team
+
+notes:
+%%
+- Tell them what you're going to tell them
+- Tell them
+- Tell them what you told them
+%%
+Hi friends my name is Tris and this is No Boilerplate, focusing on fast, technical videos.
+
+All good teams are alike; each bad team is bad in its own way.
+%% to paraphrase Tolstoy %%
+
+Software is an incredible thing, isn't it?
+Combined with the internet, a small team of friends can change the world overnight.
+
+Every company, no matter what their industry, must now run a tech team, even if only to maintain their website.
+
+So why are they all so bad at it?
+
+---
+
+![[cc-logo.png]]
+
+## Public Domain Videos
+
+[https://github.com/0atman/noboilerplate/](https://github.com/0atman/noboilerplate/)
+
+notes:
+Everything you see in this video: script, links, and images are part of a plain-text markdown document available freely on GitHub under a public domain licence.
+
+---
+
+# First World Problems
+
+notes:
+If you've worked in a web team, tech team or any digital creative team, you've likely felt the pain.
+
+- Bad software,
+- constantly changing processes,
+- and lots and lots of meetings.
+
+I discussed some of these problems in my Agile video that made me a lot of friends.
+But today, I want to go bigger.
+You can solve all these problems in a single blow.
+
+The secret is, in order to do more, you must have the discipline to do LESS.
+
+---
+
+<split even>
+
+![[rework-book.png|400]]
+
+&nbsp;
+&nbsp;
+&nbsp;
+&nbsp;
+&nbsp;
+&nbsp;
+
+![[remote-book.png|400]]
+
+</split>
+
+notes:
+
+A lot of the ideas that I will mention today are not new.
+They've been well-understood in the startup and digital world for a long time.
+
+But regression to the mean is prevalent.
+
+It's not just enough to argue for good tools today, you must stop the future churn of new apps and processes that solve the same things in different, but equivalent ways.
+
+And you do this with a Ulysses pact.
+
+---
+
+![[ulysses-and-the-sirens-waterhouse.jpg]]
+
+_"Ulysses and the Sirens"_ [John William Waterhouse](https://en.wikipedia.org/wiki/John_William_Waterhouse)
+
+notes:
+
+# Tie Yourself to the Mast
+
+In the Odyssey, Ulysses had to travel through siren-infested waters.
```

Comment:
This is a tough one - the term is indeed "Ulysses pact" as Karl so emphatically pointed out, but when talking about the man himself the name "Odysseus" is much more commonly understood.  That's why personally I wouldn't hate the use of "Odyssean pact". But :shrug:

Reply:
freaking nightmare this one.

Reply:
OK, done some research and fixed it: _Ulysses_ is used in English literature, including the Pact, whereas Odysseus is the historical Greek figure. I've added a quite aside to explain this confusing problem.